lib/juttle-subprocess.js

```diff
@@ -84,7 +81,7 @@ process.on('message', function(msg) {
 
         var compile_options = {
             stage: 'eval',
-            fg_processors: [implicit_views(config.implicit_view), optimize, views_sourceinfo],
+            implicit_view: config.implicit_view,
```

Comment:
Is config.implicit_view supposed to be something at the top of the juttle config file, and not something below a `juttle` property or similar?